src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -87,8 +87,9 @@
         timestamp-str]])))
 
 (defview quoted-message
-  [_ {:keys [from parsed-text image]} outgoing current-public-key public? pinned]
-  (letsubs [contact-name [:contacts/contact-name-by-identity from]]
+  [_ {:keys [from parsed-text image audio sticker id] :as message} outgoing current-public-key public? pinned chat-id]
+  (letsubs [contact-name [:contacts/contact-name-by-identity from]
+            repliedMsg (@(re-frame/subscribe [:chats/chat-messages chat-id]) id)] ;; Msg replied to
```

Comment:
it probably makes sense to stop using `defview`/`letsubs` here

Reply:
and better to use get `(get @(re-frame/subscribe [:chats/chat-messages chat-id]) id)`

Reply:
I think `repliedMsg`is not following conventions. It should probably be `replied-msg`, and consider using `replied-message` instead as the character saving adds no value imo.

Reply:
@flexsurfer but I need to have the replied-message in a variable to use later on in the `if` condition, and when rendering the reply view. How can I do that with `(get @(re-frame/subscribe [:chats/chat-messages chat-id]) id)` ?